packages/widget-chat/src/components/list-separator/index.js

```diff
@@ -0,0 +1,29 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+
+import styles from './styles.css';
+
+function ListSeparator(props) {
+  const mainStyles = [`separator`, styles.separator];
+  const textStyles = [`separator-text`, styles.separatorText];
+  if (props.classNames) {
+    props.classNames.forEach((className) => {
+      mainStyles.push(className);
+      mainStyles.push(styles[className]);
+      textStyles.push(className);
+      textStyles.push(styles[className]);
+    });
```

Comment:
It also looks like `ListSeparator` is only ever normal or for new messages gets the classNames [`upper`, `blue`]. 
I'm not a fan of allowing for arbitrary classnames like this. The context and styles for this should come from the parent that is passing the class and that should be where it's set. Maybe instead of doing this, add a wrapper and styles in the parent component instead?